### PR TITLE
Remove courses from the adminapprove table when they are deleted

### LINKED ISSUE #22: Can trigger a "Can't find data record in database table course." error
Hi team

I wanted to let you know that the step can be blocked on a "Can't find data record in database table course." error, i.e. I can't get to the adminapprove table any more.

grafik

You once asked me if that had to do with a course being removed in the meantime. I think: maybe yes!
If you advise me how I could check this I will gladly do this.
Anyway this can be happening at any time when an admin or manager remove a course manually so I think preventing that error from happening codewise would still be the best thing to do.

Best,
Luca

### PR DESCRIPTION
**Pull request by justusdieckmann**
Depends on  learnweb/moodle-tool_lifecycle#117
Fixes #22 
(Also fixes Travis)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,92 +1,84 @@
...
     - $HOME/.composer/cache
     - $HOME/.npm
 
 addons:
-  postgresql: "9.4"
-  firefox: "47.0.1"
-  apt:
-    packages:
-      - chromium-chromedriver
+  postgresql: "9.5"
```
Comment (sevenseas-gists) [inline .travis.yml]: current master requires 9.6

**Review by sevenseas-gists — CHANGES_REQUESTED** [review]

File: lib.php [hunk 2]
```diff
@@ -24,6 +24,7 @@
 
 namespace tool_lifecycle\step;
 
+use tool_lifecycle\local\entity\process;
```
Comment (sevenseas-gists) [inline lib.php:27]: is this being used? I only see $process, might be an accidental autostatement from IDE?

Comment (justusdieckmann) [reply lib.php:27]: Yes, in Line 113 in the documentation.

Comment (sevenseas-gists) [reply lib.php:27]: Ah, I didn't consider it being used in the documentation

**Review by sevenseas-gists — APPROVED** [review]

**Pull request merged**